### ISSUE TITLE
bug(MatSelect): MatSelect does not read out options to VoiceOver when using keyboard navigation

### ISSUE THREAD

**Issue by wildcardalice** [angular/components#28694]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

When using a screenreader to interact with a select-based form field, the options are not always read out. I notice that focus remains on the mat-select itself when you use the keyboard to activate it (see https://i.imgur.com/MwKPWxW.png). I would expect this focus to be on the options panel instead.

### Reproduction

Steps to reproduce:
1. Navigate to https://v5.material.angular.io/components/select/examples 
2. Use the keyboard to navigate to an example dropdown, such as the "Select with form field features"
3. Activate the dropdown using the keyboard, and attempt to get a readout of the options

### Expected Behavior

The screenreader should read out all options, or provide a way to read all options

### Actual Behavior

I am unable to get the screen reader to read out dog, cat, cow, or fox. The only option I can read out is the first option ("In text -- not selected") by using ctrl+opt+shift+down, but continued navigation does not shift focus or read additional options.

### Environment

- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): macOS
- Screen Reader: VoiceOver

**Comment by amysorto** [comment]
Unfortunately v5 refers to our legacy components, and a change is unlikely. I would highly recommend upgrading to our new components (starting in v15), a lot of them have a lot of accessiblity improvements compared to our legacy ones.

**Comment by wildcardalice** [comment]
Followed up here: https://github.com/angular/components/issues/28709
